mythril/mythril/mythril_analyzer.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""mythril.py: Bug hunting on the Ethereum blockchain
+
+   http://www.github.com/b-mueller/mythril
```

Comment:
While we're at it, we can probably change this link to https://github.com/ConsenSys/mythril-classic

Reply:
Sounds good.